dev/kyuubi-extension-spark-3-1/pom.xml

```diff
@@ -27,9 +27,9 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>kyuubi-extension-spark_3.1</artifactId>
+    <artifactId>kyuubi-extension-spark-3-1</artifactId>
```

Comment:
Name the module by minimal compatible version, e.g. 
`elasticsearch-spark-13` Elasticsearch Spark (for Spark 1.3 1.6)
`elasticsearch-spark-20` Elasticsearch Spark (for Spark 2.X)

Reply:
`spark-3-1` seems strange, `spark-3_1` ? or just `spark_31`

Reply:
we should avoid using `_`.`3-1` match the style of `spark-sql-kafka-0-10`, I'm fine with both `3-1` and `31`